Dockerfile

```diff
@@ -36,7 +40,8 @@ RUN \
 # Note that we are not using --platform here, so it will choose the base image for the target platform, not the build platform.
 # By using "distroless/static" instead of "distroless/static-debianXX" we can float on the latest stable version of debian.
 # See https://github.com/GoogleContainerTools/distroless#base-operating-system
-FROM gcr.io/distroless/static:nonroot@sha256:2a9e2b4fa771d31fe3346a873be845bfc2159695b9f90ca08e950497006ccc2e
+FROM gcr.io/distroless/static:nonroot@sha256:2a9e2b4fa771d31fe3346a873be845bfc2159695b9f90ca08e950497006ccc2e as default_distroless
```

Comment:
I didn't bump this.
I noted in chat that Dependabot only watches the first FROM, it doesn't check the rest.  So our `gcr.io/distroless/static:nonroot@sha256:2a9e2b...` image is actually a bit out of date.  We could bump it in this PR as well.

Reply:
Good catch. When I pull that image on my laptop, I get `sha256:91ca4720011393f4d4cab3a01fa5814ee2714b7d40e6c74f2505f74168398ca9`.

Dependabot has previously bumped this image for us. See https://github.com/vmware-tanzu/pinniped/pulls?q=is%3Apr+distroless+is%3Aclosed+author%3Aapp%2Fdependabot.

Any idea why it stopped?

Reply:
Not sure yet. Fishing around [the docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file?learn=dependency_version_updates&learnProduct=code-security#docker), etc.